src/components/PostcodeLookup/PostcodeLookup.js

```diff
@@ -72,9 +72,8 @@ class PostcodeLookup extends Component {
         this.fieldRefs = refs;
       }
     };
-
-    this.addressLookup = this.addressLookup.bind(this);
     this.showAddressFields = this.showAddressFields.bind(this);
+    this.plusURL = 'https://lookups.sls.comicrelief.com/postcode/lookup?query=';
```

Comment:
I wonder if having this hardcoded is wise.. maybe better as an env variable?

Reply:
@AndyEPhipps probably true, however we will have to put this in the applications pipelines using plus and therefore wouldn't change as part of this PR.

Our backups for this will always use the same url, so not sure whether we should change before campaign.